src/main/java/fr/theorozier/webstreamer/display/render/DisplayLayerManager.java

```diff
@@ -24,6 +24,9 @@ public class DisplayLayerManager extends DisplayLayerMap<URI> {
 
     /** Time in nanoseconds (monotonic) of the last cleanup for unused layers. */
     private long lastCleanup = 0;
+
+    /** Used to handle SVG images. **/
+    private final DisplayLayerSVGMap svgMap = new DisplayLayerSVGMap(this.res); //TODO: probably change where this happens.
```

Comment:
Not needed, see `getNewLayer`

Reply:
Resolved in https://github.com/mindstorm38/fabric-webstreamer/pull/28/commits/1c0c2e9fdf6e0900dbf82b5f743fdc32ea4a4e03